<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContadordeCristian</class>
 <widget class="QWidget" name="ContadordeCristian">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cristian's Counter</string>
  </property>
  <widget class="QPushButton" name="buttonPause">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Pause counter</string>
   </property>
   <property name="text">
    <string> Pause</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pause.png</normaloff>pause.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
   <property name="digitCount">
    <number>6</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Filled</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonStart">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Start counting</string>
   </property>
   <property name="text">
    <string> Start</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>start.png</normaloff>start.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSetPeriod">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set period (millis)</string>
   </property>
   <property name="text">
    <string>Set period (ms)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePeriod">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>471</width>
     <height>27</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0,0,0);</string>
   </property>
   <property name="text">
    <string>Author: Cristian VÃ¡zquez Cordero - Subject: Robotics.</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineError">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0,0,0);color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineOk">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0,0,0);color: green;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRestart">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Restart counter</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>restart.png</normaloff>restart.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
